tf-modules/archive/ems_reporting.tf

```diff
@@ -1,6 +1,7 @@
 # EMS Distribution Report
 
 resource "aws_lambda_function" "ems_distribution_report" {
+  count            = var.ems_deploy ? 1 : 0
```

Comment:
And what are these counts used for?

Reply:
Not this PR, but we really need to move to TF 0.13: 

`Terraform 0.13 supports count on both resource and module blocks. Prior versions only supported it on resource blocks.` (https://learn.hashicorp.com/tutorials/terraform/count

Setting a single count on all these resources versus just abstracting it into a seperate submodule doesn't feel great.  

Again, not this PR, just noting it for future us.